contracts/msig_court/lib.rs

```diff
@@ -0,0 +1,538 @@
+#![cfg_attr(not(feature = "std"), no_std, no_main)]
+
+#[cfg(test)]
+mod mock;
+
+#[ink::contract(env = liberland_extension::LiberlandEnvironment)]
+mod msig_court {
+	use ink::codegen::Env;
+	use ink::prelude::vec::Vec;
+	use ink::storage::Mapping;
+	use liberland_extension::LLMForceTransferArguments;
+
+	#[derive(Debug, Clone, PartialEq, Eq)]
+	#[ink::scale_derive(Encode, Decode, TypeInfo)]
+	#[cfg_attr(feature = "std", derive(ink::storage::traits::StorageLayout))]
+	pub enum Proposal {
+		LLMForceTransfer(LLMForceTransferArguments),
+		SetGovernance { threshold: u32, judges: Vec<AccountId> },
```

Comment:
There's no such thing as sudo/root for contracts. So instead, adding/removing judges can be made by current judges, just like any other proposal.

If that's not OK and we want sudo, we can add a contract owner, set it to some address with no private key then use sudo.sudoAs to trigger it. Let me know what you prefer and I'll adjust if needed.

Reply:
Can senate or congress be contract owner (for now)? I expect making that tx would be cringe but doable...

Reply:
Right now they can't. We can use custom-account pallet for this, but we need to allow contract interactions in the call filter. It's trivial to do.
Should I add this? Do we want senate or congress? If congress, can senate veto?